tntconcept-rest/src/test/kotlin/com/autentia/tnt/api/binnacle/SearchControllerTest.kt

```diff
@@ -20,25 +20,25 @@ internal class SearchControllerTest {
     private val searchController = SearchController(searchUseCase)
 
     @Test
-    fun `return empty lists for unknown role ids`() {
```

Comment:
Este test unitario se puede borrar, mantenemos el de IT (ver que se ha borrado en la misma PR ProjectControllerTest)

Reply:
deleted!